pkg/server/plugin/nodeattestor/tpmdevid/devid.go

```diff
@@ -0,0 +1,491 @@
+package tpmdevid
+
+import (
+	"context"
+	"crypto/rsa"
+	"crypto/sha1" //nolint: gosec // SHA1 use is according to specification
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/hashicorp/hcl"
+	"github.com/spiffe/spire/pkg/common/catalog"
+	"github.com/spiffe/spire/pkg/common/idutil"
+	common_devid "github.com/spiffe/spire/pkg/common/plugin/tpmdevid"
+	"github.com/spiffe/spire/pkg/common/util"
+	spc "github.com/spiffe/spire/proto/spire/common"
+	spi "github.com/spiffe/spire/proto/spire/common/plugin"
+	nodeattestorv0 "github.com/spiffe/spire/proto/spire/plugin/server/nodeattestor/v0"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func BuiltIn() catalog.BuiltIn {
+	return builtin(New())
+}
+
+func builtin(p *Plugin) catalog.BuiltIn {
+	return catalog.MakeBuiltIn(common_devid.PluginName,
+		nodeattestorv0.NodeAttestorPluginServer(p),
+	)
+}
+
+type config struct {
+	trustDomain string
+
+	devIDRoots          *x509.CertPool
+	ekRoots             *x509.CertPool
+	checkDevIDResidency bool
+}
+
+type Config struct {
+	DevIDBundlePath       string `hcl:"devid_bundle_path"`
+	EndorsementBundlePath string `hcl:"endorsement_bundle_path"`
+	CheckDevIDResidency   bool   `hcl:"check_devid_residency"`
```

Comment:
This configurable is undocumented, I think it is left over from the previous CR?